LibreNMS/Util/Html.php

```diff
@@ -185,4 +186,18 @@ public static function powerStateLabel($state): array
                 return ['UNKNOWN', 'label-default'];
         }
     }
+
```

Comment:
You can add functions to enum classes. Might be a better central location. Not sure ?